Makefile

```diff
@@ -151,8 +151,19 @@ ${KOPS}:
 .PHONY: codegen
 codegen:
 	go install k8s.io/kops/upup/tools/generators/...
-	${GOPATH_1ST}/bin/fitask --input-dirs k8s.io/kops/upup/pkg/fi/... \
-	    --go-header-file "hack/boilerplate/boilerplate.generatego.txt"
+	${GOPATH_1ST}/bin/fitask \
+		--input-dirs k8s.io/kops/upup/pkg/fi/... \
+		--go-header-file hack/boilerplate/boilerplate.generatego.txt \
+		--output-base ${GOPATH_1ST}/src/k8s.io/kops/
```

Comment:
We're trying to reduce our reliance on `GOPATH`, could we use `KOPS_ROOT` instead? It would probably be good to compile the fitask binary into `_output/bin` to match the `TOOLS_BIN` references in `hack/`.

Reply:
Good point. Done.